pkg/features/features.go

```diff
@@ -9,10 +9,18 @@ import (
 const (
 	// CraneAutoscaling enables the autoscaling features for workloads.
 	CraneAutoscaling featuregate.Feature = "Autoscaling"
+
+	// CraneNodeResource enables the node resource features.
+	CraneNodeResource featuregate.Feature = "NodeResource"
+
+	// CraneClusterNodePrediction enables the cluster node prediction features.
+	CraneClusterNodePrediction featuregate.Feature = "ClusterNodePrediction"
 )
 
 var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	CraneAutoscaling: {Default: true, PreRelease: featuregate.Alpha},
+	CraneAutoscaling:           {Default: true, PreRelease: featuregate.Alpha},
+	CraneNodeResource:          {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
How about merge NodeResource and ClusterNodePrediction into one feature gate? If so, we need to use a appropriate name for it.